src/SIM/SIM2D.C

```diff
@@ -81,8 +81,9 @@ bool SIM2D::parseGeometryTag (const TiXmlElement* elem)
     if (utl::getAttribute(elem,"lowerpatch",lowpatch))
       uppatch = myModel.size();
     utl::getAttribute(elem,"upperpatch",uppatch);
+    int nPatch = nGlPatches > 0 ? nGlPatches : myModel.size();
 
-    if (lowpatch < 1 || uppatch > (int)myModel.size())
+    if (lowpatch < 1 || uppatch > nPatch)
```

Comment:
Når er det at nGlPatches er lik 0 egentlig? For seriell kjøring blir den vel satt lik myModel.size() slik at nPatch-variabelen her og ellers burde være overflødig (men det er kanskje noe jeg har oversett, det burde i såfall vært dokumentert i koden ett sted hvorfor det er slik)